src/Box/Box.cs

```diff
@@ -113,7 +113,6 @@ void OnSimulationEnded()
 				GlobalPosition = initialPos;
 			}
 
-			Rotation = Vector3.Zero;
```

Comment:
Like Robert said, it's not redundant. The Box's Rotation needs to be set back to its original value. However, there is a bug. Zero might not be the original Rotation.

Changing `GlobalPosition = initialPos` to `GlobalTranform = initialTransform` would fix that.

Reply:
The same fix also need to be applied to Pallet.cs.